### PR TITLE
Symplectic

### PR DESCRIPTION
**Pull request by dalonsoa**
Instructions to add software to Symplectic Elements, at least in its Imperial College Implementation.

### CODE REVIEW

File: symplectic.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+After logging into Symplectic, you will go straight to your "**Home**", which shows a summary of your publications, grants and gives you a couple of choices to deposit your work after this has been published.
+
+![Image](SymplecticImages/ProfileScreen.png)
+
+None of these options are useful for software, but just for journal or conference publications. To add a software, go to the "**Menu**" tab at the top, and then to "**Publications**", under the "**Manage**" card on the left hand side. 
+
+![Image](SymplecticImages/GoToPublications.png)
+
+This will bring you to you complete publications list, showing those that you have claimed, those that are pending and those that you have rejected - possibly because they have been wrongly identified as yours by the system. On this screen, click on "**Add a new publication**" on the top-right corner:
```
Comment (olexandr-konovalov) [inline symplectic.md]: "you to you" -> "you to your"

File: symplectic.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+title: Code4REF
+---
+
+# How to report software outputs in Symplectic Elements
+
+This guide shows the method for recording software outputs in Symplectic Elements as implemented at Imperial College London (16th October 2019).
+
+After logging into Symplectic, you will go straight to your "**Home**", which shows a summary of your publications, grants and gives you a couple of choices to deposit your work after this has been published.
```
Comment (olexandr-konovalov) [inline symplectic.md]: "after this has been published" -> "after it has been published"

File: symplectic.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+None of these options are useful for software, but just for journal or conference publications. To add a software, go to the "**Menu**" tab at the top, and then to "**Publications**", under the "**Manage**" card on the left hand side. 
+
+![Image](SymplecticImages/GoToPublications.png)
+
+This will bring you to you complete publications list, showing those that you have claimed, those that are pending and those that you have rejected - possibly because they have been wrongly identified as yours by the system. On this screen, click on "**Add a new publication**" on the top-right corner:
+
+![Image](SymplecticImages/AddNewPublication.png)
+
+A very large list of outcomes to add opens - good to remind you of other things that you could (should) be submitting - and among them, "**Add a new software / code**".
```
Comment (olexandr-konovalov) [inline symplectic.md]: "list of outcomes" -> "list of types of outcomes" or "list of kinds of outcomes"?

File: symplectic.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+![Image](SymplecticImages/AddNewSoftware.png)
+
+When clicking in that option, the process of adding a new software starts:
```
Comment (olexandr-konovalov) [inline symplectic.md]: "in that option" -> "on that option" (or "selecting that option")

**Review by olexandr-konovalov — APPROVED** [review]
Thank you @dalonsoa - excited to get one more system covered! I have left several minor suggestions.

**Comment by dalonsoa** [comment]
Done! My pleasure to contribute to this great initiative :)

**Comment by olexandr-konovalov** [comment]
Perfect, thanks @dalonsoa !

**Comment by olexandr-konovalov** [comment]
Now live at https://code4ref.github.io/symplectic.html

**Pull request merged**